### ISSUE TITLE
File changes (unsaved) indicator is not present

### ISSUE THREAD

**Issue by sid8491** [getcursor/cursor#1132]
In VS Code, if we make changes there is an indicator like this:
<img width="450" alt="Screenshot 2023-12-15 at 1 29 22 PM" src="https://github.com/getcursor/cursor/assets/8565062/73905529-2825-4071-b457-76881b0c8d36">

But this is not present in the Cursor:
<img width="394" alt="Screenshot 2023-12-15 at 1 30 08 PM" src="https://github.com/getcursor/cursor/assets/8565062/b067d454-e2e3-4fd5-8eeb-963ef25f0b40">

I am not sure its a bug or unimplemented feature.

**Comment by truell20** [comment]
Does this always happen for you? Is it easy to reproduce?

**Comment by sid8491** [comment]
> Does this always happen for you? Is it easy to reproduce?

yes, it's happening always. I have not seen the indicator even once.
is there some setting to enable/disable this?

**Comment by diegodorado** [comment]
i got a related issue: if i change a file i do get the dirty indicator, but if i undo the changes so the buffer has the same contents as the file, the dirty indicator is still present
